src/components/calcite-time-picker/calcite-time-picker.tsx

```diff
@@ -280,14 +298,24 @@ export class CalciteTimePicker {
   //
   // --------------------------------------------------------------------------
 
+  private buttonActivated(event: KeyboardEvent): boolean {
+    const key = getKey(event.key);
+
+    if (key === " ") {
```

Comment:
Nitpick: If it's an activation key, I think you can call `preventDefault` regardless of the key.